genai-perf/genai_perf/config/input/config_command.py

```diff
@@ -112,6 +121,8 @@ def _parse_model_names(self, model_names: Any) -> None:
     def _infer_settings(self) -> None:
         self.endpoint.infer_settings(model_name=self.model_names[0])
         self.input.infer_settings()
+        self.perf_analyzer.infer_settings()
+        self.tokenizer.infer_settings(model_name=self.model_names[0])
```

Comment:
What is this magic number?

Reply:
Not sure how to un-magic this! The user provides a list of model names, but we only ever care about the first model name they provide except in very specific instances.

Reply:
Btw, this magic [0] occurs throughout the GAP code and it's probably beyond the scope of this story to fix it.

Reply:
Yeah I think this is fine since we don't have a good way of handling multiple model names (or multi-lora) atm.